<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_central">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <item>
       <widget class="QPushButton" name="pushButton_switchWindow">
        <property name="text">
         <string>交换机信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_switchWindow_3">
        <property name="text">
         <string>文件查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="fileSaveGroupBox">
      <property name="title">
       <string>File Save</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_save">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_path">
         <item>
          <widget class="QLabel" name="label_savePath">
           <property name="text">
            <string>File Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_savePath"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_buttons">
         <item>
          <widget class="QPushButton" name="pushButton_browse">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_browse_2">
           <property name="text">
            <string>存储</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_status">
      <item>
       <widget class="QLabel" name="label_time">
        <property name="text">
         <string>当地时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_switchStatus">
        <property name="text">
         <string>连接状态: Linking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
